web/files/referencedby.go

```diff
@@ -10,6 +11,36 @@ import (
 	"github.com/labstack/echo/v4"
 )
 
+// ReferencesDocument is JSON-API document, identified by the mediatype
+// application/vnd.api+json
+// See http://jsonapi.org/format/#document-structure
+// It differs from jsonapi.Document because it represents a list of relationships
```

Comment:
I don't understand. In my mind, the response should be a `jsonapi.Document`. And a `jsonapi.Document` can have a list as its `data`, so I don't see the difference with the `ReferencesDocument`.

Reply:
You're right about the `data` field. But in `Document`, the `meta` field is a `RelationshipMeta` struct, with a `count`, not a `rev`.
Besides, when I looked at the methods in `jsonapi`, most of them were filling the `data` field with an `Object` which has a `type` and `attributes` and I wasn't sure this would fit our needs.

I should probably have asked you.